packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts

```diff
@@ -28,25 +30,29 @@ type TokenType = 'ETH' | 'ERC-20'
 type FastBridgeName = `${FastBridgeNames}`
 type NonCanonicalTokenName = `${NonCanonicalTokenNames}`
 
-const FathomNetworkNames = ['Arbitrum One', 'Arbitrum Nova'] as const
+const AnalyticsNetworkNames = ['Arbitrum One', 'Arbitrum Nova'] as const
 type AllNetworkNames = ReturnType<typeof getNetworkName>
-type FathomNetworkName = (typeof FathomNetworkNames)[number]
-export const isFathomNetworkName = (
+type AnalyticsNetworkName = (typeof AnalyticsNetworkNames)[number]
+export const isAnalyticsNetworkName = (
   networkName: AllNetworkNames
-): networkName is FathomNetworkName => {
-  return FathomNetworkNames.includes(networkName as FathomNetworkName)
+): networkName is AnalyticsNetworkName => {
+  if (process.env.NODE_ENV === 'development') {
+    // sends events for any network when in dev
```

Comment:
Even though we are not sending any events in dev, it's good to have for debugging